src/client/utils/property-overriding.ts

```diff
@@ -54,3 +55,53 @@ export function overrideDescriptor<O extends object, K extends keyof O> (obj: O,
 
     nativeMethods.objectDefineProperty(obj, prop, descriptor);
 }
+
+function overrideFunctionName (fn: Function, name: string) {
+    const wrapperNameDescriptor = nativeMethods.objectGetOwnPropertyDescriptor(fn, 'name');
+
+    if (wrapperNameDescriptor) {
+        // eslint-disable-next-line no-restricted-properties
+        wrapperNameDescriptor.value = name;
+
+        nativeMethods.objectDefineProperty(fn, 'name', wrapperNameDescriptor);
+    }
+}
+
+export function overrideStringRepresentation (nativeFnWrapper: Function, nativeFn: Function): void {
```

Comment:
The name of this function is unclear.
Instead of it use combination of  the `overrideFunctionName` and `overrideToString` functions.